lib/ipmi_backend_utils.pm

```diff
@@ -146,7 +147,19 @@ sub umount_installation_disk {
 }
 
 sub get_installation_partition {
-    my $partition = script_output("fdisk -l | grep \"^\/dev\/sda\.\*\\\*\" | cut -d ' ' -f 1");
+    my $partition = '';
+    if (sle_version_at_least('15')) {
+        my $y2log_file                = '/var/log/YaST2/y2log';
+        my $root_partition_commit_msg = script_output("grep \"Commit Action \\\"Adding mount point / of .* to /etc/fstab\" $y2log_file");
+        $root_partition_commit_msg =~ /Commit Action \"Adding mount point \/ of ([^\s]*) to \/etc\/fstab/m;
+        $partition = $1;
```

Comment:
If you still think this is a good idea as this method `get_installation_partition` is getting more complicated I suggest to add a doc string above the method with an explanation about what is the goal.

Reply:
ok, will add comment in code